### PR TITLE
Remove logic to save artifacts to old gcs location.

### PR DESCRIPTION
**Pull request by godofredoc**
This is to finalize the process of the bucket migration to a new
location.

Bug: https://github.com/flutter/flutter/issues/75363

## Pre-launch Checklist

- [X] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [X] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [X] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [X] I signed the [CLA].
- [X] I listed at least one issue that this PR fixes in the description above.
- [X] I updated/added relevant documentation (doc comments with `///`).
- [X] I added new tests to check the change I am making or feature I am adding, or Hixie said the PR is test-exempt.
- [X] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

File: dev/bots/prepare_package.dart [hunk 1]
```diff
@@ -577,28 +574,26 @@ class ArchivePublisher {
...
-    for (final bool isNew in <bool>[false, true]) {
-      final String releaseFolder = isNew ? newGsReleaseFolder : oldGsReleaseFolder;
-      final String destGsPath = '$releaseFolder/$destinationArchivePath';
-      if (!forceUpload) {
-        if (await _cloudPathExists(destGsPath) && !dryRun) {
-          throw PreparePackageException(
-            'File $destGsPath already exists on cloud storage!',
-          );
-        }
+    //final String releaseFolder = gsReleaseFolder;
```
Comment (christopherfujino) [inline dev/bots/prepare_package.dart:577]: delete?

File: dev/bots/prepare_package.dart [hunk 2]
```diff
@@ -577,28 +574,26 @@ class ArchivePublisher {
...
+    );
+    assert(tempDir.existsSync());
+    await _updateMetadata('$gsReleaseFolder/${getMetadataFilename(platform)}');
   }
 
-  Future<Map<String, dynamic>> _addRelease(Map<String, dynamic> jsonData, {bool newBucket=true}) async {
-    final String tmpBaseUrl = newBucket ? newBaseUrl : oldBaseUrl;
-    jsonData['base_url'] = '$tmpBaseUrl$releaseFolder';
+  Future<Map<String, dynamic>> _addRelease(Map<String, dynamic> jsonData) async {
+    //final String tmpBaseUrl = baseUrl;
```
Comment (christopherfujino) [inline dev/bots/prepare_package.dart:595]: delete?

**Review by christopherfujino — APPROVED** [review]
Two nit comment deletes, otherwise LGTM

File: dev/bots/prepare_package.dart (hunk 1 above)
Comment (godofredoc) [reply dev/bots/prepare_package.dart:577]: done!

File: dev/bots/prepare_package.dart (hunk 2 above)
Comment (godofredoc) [reply dev/bots/prepare_package.dart:595]: done!

**Pull request merged**